bundles/org.openhab.binding.lcn/src/main/java/org/openhab/binding/lcn/internal/LcnModuleActions.java

```diff
@@ -125,6 +125,11 @@ public void sendDynamicText(
                 text = new String();
             }
 
+            // some LCN-GTxD don't display the text if it fits exactly in one chunk. Observed with GT10D 8.0.
+            if (text.getBytes(LcnDefs.LCN_ENCODING).length % DYN_TEXT_CHUNK_LENGTH == 0) {
```

Comment:
The default charset varies from system to system so it is better to specify it yourself instead of using the default.

Reply:
`LcnDefs.LCN_ENCODING` is set to `StandardCharsets.UTF_8`.

Reply:
...I have no idea why I didn't notice that.